docs/kb/consistency.rst

```diff
@@ -0,0 +1,91 @@
+
+ScyllaDB, as well as `Apache Cassandra <https://cassandra.apache.org/>`_ and other NoSQL databases, sacrifice a degree of consistency in order to 
+increase availability. Rather than providing strong consistency, they provide `eventual consistency <https://www.scylladb.com/glossary/eventual-consistency/>`_. This means that in some cases, a read request will fail to return the result of the latest WRITE. 
+
+In ScyllaDB (and Apache Cassandra), consistency is tunable - for a given query, the client can specify a `Consistency Level <https://university.scylladb.com/courses/scylla-essentials-overview/lessons/architecture/topic/consistency-level-cl/>`_.
+A Consistency Level is the number of replicas required to respond to a request for the request to be considered successful.  The possibility to
+specify a Consistency Level per query is known as `Tunable Consistency <https://docs.scylladb.com/stable/glossary.html#term-Tunable-Consistency>`_. 
+
+A database using the ACID model expects a unit of work to be atomic. It is all-or-nothing. While that unit of work is being done, the records or 
+tables affected may be locked. 
+
+On the other hand, distributed databases with a BASE model trade off strong consistency with high availability. The records stay available, 
+but once the query has been completed across a majority of `nodes <https://university.scylladb.com/courses/scylla-essentials-overview/lessons/architecture/topic/node/>`_, 
+or as defined by the Consistency Level, the transaction is deemed successful. Data replication across all nodes can take some time, but the data in all 
+the nodes will become consistent eventually. Once the query is deemed successful, queries for the data will consistently provide the updated data, 
+even before data replication reaches the last few nodes.
+
+Early results of eventual consistency data queries may not have the most recent updates. This is because it takes time for updates to reach replicas 
+across a distributed database cluster. In strong consistency, however, data is sent to every replica the moment a query is made. This causes delays 
+because responses to any new requests must wait while the replicas are updated. 
+
+To summarize, in contrast to SQL’s ACID guarantees, NoSQL databases generally provide BASE guarantees. This enables high availability and relaxes 
+the stringent consistency.
+
+A related feature is `Lightweight Transactions <https://docs.scylladb.com/stable/using-scylla/lwt>`_ (LWT). LWT in ScyllaDB allow the client 
+to modify data based on its current state: that is, to perform an update that is executed only if a row does not exist or contains a certain value. 
+LWT are limited to a single conditional statement, which allows an “atomic compare and set” operation. That is, it checks if a condition is true, 
+and if so, it conducts the transaction. If the condition is not met, the transaction does not go through.
```

Comment:
Again, this paragraph is 100% true, but it's not clear how it relates to the rest of the text. Are LWT statements "consistent", "strongly consistent", or what?